lib/src/configuration.dart

```diff
@@ -39,14 +39,23 @@ class Configuration {
   /// [readOnly] controls whether a [Realm] is opened as readonly.
   /// This allows opening it from locked locations such as resources,
   /// bundled with an application.  The realm file must already exists.
-  Configuration(List<SchemaObject> schemaObjects, {bool readOnly = false})
+  ///
+  /// [inMemory] specifies if this Realm should be opened in-memory.
+  /// This still requires a path (can be the default path) to identify the [Realm] so other processes can open the same [Realm].
```

Comment:
Since we document this below I suggest we keep it short here. as the fifo api doc. So to remove these comments

Reply:
done